README.md

```diff
@@ -31,61 +31,65 @@ If you do not have [Composer](http://getcomposer.org/), you may install it by fo
 
-See https://youtu.be/jRlinjpTl0c how to video for the whole process of this command usage.
+See [a how-to video](https://youtu.be/jRlinjpTl0c) for the whole process of this command usage.
 
+### Development environment
 
-## Development environment
+To get an environment that was configured especially for development with the distribution, remove `--no-dev` from the composer command.
 
-You should use composer command without `--no-dev` if you would like to get environment that was configured especially for OpenY. This means you'd remove Vagrant/Docksal from the code tree.
 So it should look like this:
 
 ```
 composer create-project ycloudyusa/yusaopeny-project:dev-main-development MY_PROJECT --no-interaction
 cd MY_PROJECT
 ```
 
-See https://youtu.be/jRlinjpTl0c how to video for the whole process of this command usage.
-=======
-
-
-### CIBox VM
-[CIBox VM](http://cibox.tools) allows you to make a contribution into OpenY in a few minutes. Just follow steps and then you'll know how to do it.
-
-- [Pre Requirements](https://github.com/ymcatwincities/openy-cibox-vm#pre-requirements)
-- [Installation](https://github.com/ymcatwincities/openy-cibox-vm#usage)
-- [Local build](https://github.com/ymcatwincities/openy-cibox-vm#reinstall-options)
-  
-Read more details on [CIBox VM](https://github.com/ymcatwincities/openy-cibox-vm) repo.
+See [a how-to video](https://youtu.be/jRlinjpTl0c) for the whole process of this command usage.
 
-### Docksal
+#### Docksal
 [Docksal](http://docksal.io) is a tool for defining and managing development environments.
 
 - [How to develop](https://github.com/ymcatwincities/openy-docksal#how-to-develop)
-- [How to run behat tests](https://github.com/ymcatwincities/openy-docksal#how-to-run-behat-tests)
   
 Read more details on [Docksal](https://github.com/ymcatwincities/openy-docksal) repo.
 
-# Use Fork for the development
+### Drupal install
 
-All development happens in the [Website Services Drupal 9/10 installation profile](https://github.com/ycloudyusa/yusaopeny). In order to start development:
+The distribution has a full user interface for step-by-step installation. Simply visit your site after building your dev environment to be taken through the installation process.
 
-1. Create fork of [Website Services installation profile](https://github.com/YCloudYUSA/yusaopeny)
+For developers who want a kick-start, you can pass many configuration options in the install process through to [drush site:install](https://www.drush.org/12.x/commands/site_install/).
+
+For example, a "complete" install with the "Carnation" theme might look like:
+
+```shell
+drush -vy si openy openy_configure_profile.preset=complete openy_theme_select.theme=openy_carnation openy_terms_of_use.agree_openy_terms=1 install_configure_form.enable_update_status_emails=NULL --account-name=admin --site-name='YMCA Website Services sandbox'
```

Comment:
Let's also add password in here, because installation takes ~40 minutes and waiting that interaction could lead to missing a moment, computer goes into sleep mode and installation breaks